botbuilder-instumentation/Managers/SentimentManager.cs

```diff
@@ -60,7 +60,13 @@ public SentimentManager(string textAnalyiticsApiKey,string textAnalyticsMinLengt
             return sentimentInfo.Documents[0].Score;
         }
 
-        private async Task<BatchResult> GetSentiment(string apiKey, string jsonSentimentInput)
+        /// <summary>
+        /// Method is marked as public virtual so that we can mock it during performance tests
+        /// </summary>
+        /// <param name="apiKey"></param>
+        /// <param name="jsonSentimentInput"></param>
+        /// <returns></returns>
+        public virtual async Task<BatchResult> GetSentiment(string apiKey, string jsonSentimentInput)
```

Comment:
If testing is the goal, it makes more sense to have the SentimentManager implement an interface, so it can be testable as well. something like `ISentimentManager`, and this will remove the need to expose this method as public, because there is no need for it be public. We will be able to tell Moq to mock the interface